handlers/hook.go

```diff
@@ -41,7 +41,7 @@ type Payload struct {
 
 func signBody(secret, body []byte) []byte {
 	computed := hmac.New(sha1.New, secret)
-	computed.Write(body)
+	_, _ = computed.Write(body)
```

Comment:
I'm wondering why we explicitly ignore errors here instead of doing what is recommended - checking the error and fail if there is one? 😅